agents/dux/agents.js

```diff
@@ -0,0 +1,47 @@
+// TODO: IK: maybe rename this / move somewhere else
```

Comment:
yeah, can we name this `dux/invited`, or something? i reckon it's confusing as agents.